translator/cmdutil/translatorutil.go

```diff
@@ -224,7 +225,31 @@ func TranslateJsonMapToYamlConfig(jsonConfigValue interface{}) (interface{}, err
 	if err = conf.Marshal(cfg); err != nil {
 		return nil, err
 	}
-	return conf.ToStringMap(), nil
+	strMap := conf.ToStringMap()
+	RemoveTLSRedacted(strMap)
+	return strMap, nil
+}
+
+func RemoveTLSRedacted(stringMap map[string]interface{}) {
+	type Node struct {
+		isTLSParent bool
+		data        map[string]interface{}
+	}
+	root := Node{isTLSParent: false, data: stringMap}
+	queue := []Node{root}
+	// Using BFS search through string Map and find sub settings of TLS
```

Comment:
Nice!

Reply:
Mind adding a quick unit test for this func?

Reply:
The other unit test actually cover this case because, the config test will fail if there is a `REDACTED`